tests/ignite/distributed/test_launcher.py

```diff
@@ -183,7 +183,7 @@ def test_idist_parallel_spawn_n_procs_native(init_method, backend, dirname):
     nproc_per_node = 4 if "gloo" == backend else torch.cuda.device_count()
     device = "cpu" if "gloo" == backend else "cuda"
     with idist.Parallel(backend=backend, nproc_per_node=nproc_per_node, init_method=init_method) as parallel:
-        parallel.run(_test_func, ws=nproc_per_node, device=device, backend=backend, true_init_method=init_method)
+        parallel.run(_test_func, ws=nproc_per_node, device=device)
```

Comment:
@ahmedo42 Is it correct ? `_test_func(index, ws, device, backend, true_init_method)` still requres `backend=backend, true_init_method=init_method`